File: MPI/mpicheck/tests/functions/2.0/MPI_Init_thread.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    int *var_0;
    char ***var_1;
    int var_2;
    int *var_3;
    int ret;
    /* calls */
    ret = MPI_Init_thread(var_0, var_1, var_2, var_3);
    ret = PMPI_Init_thread(var_0, var_1, var_2, var_3);
    return 0;
}

File: MPI/mpicheck/tests/functions/2.0/MPI_Init_thread.f90

        
        program main
        use mpi
        
        INTEGER var_2
       INTEGER var_3
       INTEGER var_4
        call mpi_init_thread(var_2, var_3, var_4)
       call pmpi_init_thread(var_2, var_3, var_4)
        end program main
    